src/Menu/BackendMenuBuilder.php

```diff
@@ -199,14 +199,6 @@ private function createAdminMenu(): ItemInterface
             ],
         ]);
 
-        $menu->getChild('Maintenance')->addChild('Fixtures', [
```

Comment:
Why remove this one? It doesn't work yet, but we'll add it sooner or later.

Reply:
I don't think we could use migrations in Bolt.

Reply:
reverted